src/internal/netInfoConfig.ts

```diff
@@ -0,0 +1,7 @@
+import { NetInfoConfig } from './types';
+
+export const netInfoConfig: NetInfoConfig = {
+  reachabilityUrl: '',
```

Comment:
I think we need a default URL here so calling the configuration I not required to get internet reachability working.

I know we currently use Google Chrome's detection, but I actually think it's best to use something more "neutral" like the Firefox one:

http://detectportal.firefox.com

Is that accessible from China?